src/Microsoft.Framework.Runtime/Loader/DesignTimeCompilationException.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Framework.Runtime
+{
+    internal class DesignTimeCompilationException : Exception, ICompilationException
+    {
+        public DesignTimeCompilationException(IList<string> errors)
```

Comment:
Maybe we should remote the unformatted messages? This means in VS right now we won't get the same nice error page in helios